### PR TITLE
Task/actuator tt closed

### PR DESCRIPTION
**Pull request by dmoranj**
Add L1 and AV actuators to the IOTA.

### CODE REVIEW

File: test/unit/provision/synchronousButtonProvision.json [hunk 1]
```diff
@@ -28,6 +28,18 @@
...
+        {
+          "name": "interaction_type",
+          "type": "string",
+          "value": "synchronous"
+        },
+        {
+          "name": "service_id",
+          "type": "string",
+          "value": "smartcity"
+        }
```
Comment (AlvaroVega) [inline test/unit/provision/synchronousButtonProvision.json]: Other possible static attributes are: internal_id, ccid, imei, imsi, aux_interaction_type and geolocation, aren't they?

File: test/unit/provision/synchronousButtonProvision.json [hunk 2]
```diff
@@ -28,6 +28,18 @@
           "name": "op_result",
```
Comment (AlvaroVega) [inline test/unit/provision/synchronousButtonProvision.json]: op_status should not be always an attribute?

File: test/unit/contextRequests/updateContextGenericConfigurationPlain.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        {
+          "name": "sleeptime",
+          "type": "string",
+          "value": "2500"
+        }
+      ]
+    }
+  ],
+  "updateAction": "UPDATE"
+}
```
Comment (gtorodelvalle) [inline test/unit/contextRequests/updateContextGenericConfigurationPlain.json]: Final blank line appreciated ;)

File: test/unit/iotagent-thinking-things_test.js [hunk 4]
```diff
@@ -221,8 +275,8 @@ describe('Southbound measure reporting', function() {
 
         it('should update the device entity in the Context Broker with both attributes',
             utils.checkContextBroker(options));
-        it('should return a 200 OK with the current value of the configuration parameter read from the CB',
-            utils.checkResponse(options, '#STACK1#6,L1,255,129,38,0,-1$,#673495,K1,300$theCondition,'));
+        it('should return a 200OK with the current value of the configuration parameter read from the CB',
```
Comment (gtorodelvalle) [inline test/unit/iotagent-thinking-things_test.js]: `200OK` -> `200 OK` :)

File: test/unit/provision/synchronousButtonProvision.json (hunk 1 above)
Comment (dmoranj) [inline test/unit/provision/synchronousButtonProvision.json]: Yes, they are. But it doesn't matter really for the unit test. The only thing that has to be tested is that this attributes end up added to the Context Broker queries. The particular attributes to be sent are transparent to the IOTA.

File: test/unit/provision/synchronousButtonProvision.json (hunk 2 above)
Comment (dmoranj) [inline test/unit/provision/synchronousButtonProvision.json]: Same as above. The IOTA doesn't make any checks about what attributes are provisioned, that's dependant on the scenario. The important thing here is to have the initial entity created with the active attributes and the static attributes added to any updateContext operation to the CB. The particular attributes used are scenario-dependant.

**Comment by gtorodelvalle** [comment]
Once received the OK from @AlvaroVega via Skype :) , LGTM and merging ;) Thanks @dmoranj !

**Pull request merged**